ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -899,6 +1186,53 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P4S Curtain Call Debuffs',
```

Comment:
https://www.twitch.tv/videos/1268415201?t=2h27m28s

* Tank tether swap seems wrong (we don't need to swap)
* Tank tether callout has ??? in it
* Group call is wrong (I'm group 1, not 4)
* Group call happens twice

What about moving the tank callout to a future PR, and keeping this as just a simple group number?

Reply:
> Group call is wrong (I'm group 1, not 4)
Not sure how that happened, data.curtainCallGroup should have evaluated to 1 and only been set when target is you... the output[1] should evaluate to the num1... is Conditions.targetIsYou() not functioning?

> Tank tether callout has ??? in it
I guess something went wrong getting Purple healer headmarker...

> Group call happens twice
I'm not sure how the call can happen twice... the condition is target is you and you only gain the affect once?

Reply:
I'm hoping it's just something I messed up with the return lines as I can't explain what else would have caused that output. The ??? output clearly means some sort of syntax was wrong on my part.

The rest I am guessing is just the bad return line causing multiple output, but the 4 is a weird one, the log would show "12.00", which should parseFloat to 12.00 and then subtract 2, gets 10 then ceiling of that is still 10 and then /10 is 1. The only way I could see it getting 4 is that it also ran for some other party member as the tank role would always get the tankswap line.

I have simplified the trigger now. I think to do a tank swap callout it needs to know who has highest aggro, but I don't know if that is something we can get. Looking at other cactbot encounters, such feature was implemented in a9s by just tracking autos the whole fight. Not sure if that's ok to do here (could at least lock it to curtain call with condition). I think if you had been the main tank at the time you'd want to swap there when you go out to break the orb?

Reply:
Regarding the headmarkers. I think I see where I may have made a mistake. I assumed 3 headmarkers would be enough, likely need all of them as there is no guarantee on the order in the log they will appear? In other words, I checked for purple tether at 0 index, but if the 3 headmarkers it picked first did not include purple (could have been all orange for instance), then it would fail. No idea how to handle missed headmarkers here, but likely needs to keep gathering, sort once in the other trigger and just hope purple is guaranteed? Unless someone gets some logs with just 1 or 2 people alive ser what happens ha.

Reply:
Reviewing some pulls I have, not sure if it is consistent but I have a fight where 1 dps was dead prior to headmarkers, so with 7/8 up it did not distribute a green headmarker.